extensions/positron-python/python_files/posit/pinned-test-requirements.txt

```diff
@@ -1,40 +1,26 @@
-bokeh==3.1.0; python_version < '3.9'
-bokeh==3.4.3; python_version == '3.9'
-bokeh==3.6.3; python_version >= '3.10'
-fastcore==1.7.5; python_version < '3.9'
-fastcore==1.7.29; python_version >= '3.9'
-geopandas==0.13.2; python_version < '3.9'
-geopandas==1.0.1; python_version >= '3.9'
-hvplot==0.11.2 ; python_version >= '3.9'
```

Comment:
There are a few dependencies that are no longer specified at all, is that intended? E.g. hvplot, ipython

Reply:
ipython is still here; I just moved it to the bottom to match `test-requirements.txt`.

I replaced `hvplot` with `holoviews` to match `test-requirements.txt` too. But it looks like `hvplot` depends on `holoviews` so we probably should be specifying `hvplot` in both files as well. It was added in https://github.com/posit-dev/positron/pull/4503 to just the pinned file.